bundles/org.openhab.binding.easee/src/main/resources/OH-INF/thing/things.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="easee"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<bridge-type id="site">
+		<label>Easee Site</label>
+		<description>Cloud connection to an Easee site.</description>
+		<config-description-ref uri="thing-type:easee:site"/>
+	</bridge-type>
+	<thing-type id="mastercharger">
+		<label>Easee Master Charger</label>
```

Comment:
I think you need to reference bridge type here which is probably why **OhInfXmlUsageCheck** is detected by SAT:
```suggestion
		<supported-bridge-type-refs>
			<bridge-type-ref id="site"/>
		</supported-bridge-type-refs>
		<label>Easee Master Charger</label>
```

See also https://www.openhab.org/docs/developer/bindings/thing-xml.html#xml-structure-for-thing-descriptions.